gallery/src/components/NxCheckbox/NxCheckboxPage.tsx

```diff
@@ -94,6 +94,21 @@ const NxCheckboxPage = () =>
               <NxCode>label</NxCode> element.
             </NxTable.Cell>
           </NxTable.Row>
+          <NxTable.Row>
+            <NxTable.Cell>checkboxAttributes</NxTable.Cell>
+            <NxTable.Cell>
+              <NxTextLink external href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox">
+                HTML Input Checkbox Attributes
+              </NxTextLink>
+            </NxTable.Cell>
+            <NxTable.Cell>No</NxTable.Cell>
+            <NxTable.Cell>
```

Comment:
I think the API around `checkboxId` is now confusing - why have to pass in the id that way instead of as part of this prop? I'd suggest marking `checkboxId` as deprecated and making it work with either

Reply:
> I think the API around `checkboxId` is now confusing - why have to pass in the id that way instead of as part of this prop? I'd suggest marking `checkboxId` as deprecated and making it work with either

I've updated it in this commit:
4c796beb23b05878976a76dfdcf72610774430aa